src/logger.rs

```diff
@@ -160,6 +160,31 @@ pub(crate) type GooseLoggerJoinHandle =
 /// Optional unbounded sender from all GooseUsers to logger thread, if enabled.
 pub(crate) type GooseLoggerTx = Option<flume::Sender<Option<GooseLog>>>;
 
+/// Formats comma separated arguments into a csv row according to RFC 4180. Every argument has to be `Display`.
+///
+/// Specifically, this encloses all values with double quotes `"` which contain a comma, a quote or a new line.
+/// Inner quotes are doubled according to RFC 4180 2.7.
+/// The fields are joined by commas `,`, but *not* terminated with a line ending.
```

Comment:
Thanks for catching that this ends up in the docs. Please hide:

`#[doc(hidden)]`